cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControlUtils.java

```diff
@@ -149,6 +154,34 @@ public static AdminClient createAdminClient(Map<String, Object> adminClientConfi
     return AdminClient.create(adminClientConfigs);
   }
 
+  /**
+   * Close the given AdminClient with the default timeout of {@link #ADMIN_CLIENT_CLOSE_TIMEOUT_MS}.
+   *
+   * @param adminClient AdminClient to be closed
+   */
+  public static void closeAdminClientWithTimeout(AdminClient adminClient) {
+    closeAdminClientWithTimeout(adminClient, ADMIN_CLIENT_CLOSE_TIMEOUT_MS);
+  }
+
+  public static void closeAdminClientWithTimeout(AdminClient adminClient, long timeoutMs) {
```

Comment:
Duplicate code -- i.e. we have the same function for `closeKafkaZkClientWithTimeout`.